Podfile

```diff
@@ -66,6 +75,11 @@ target 'Database' do
   pod 'Firebase/Database'
 end
 
+target 'Storage' do
```

Comment:
The differences are highlighted in Xcode, which is where they're modified anyway. If we rename the dylib target, we have to rename the framework while copying it (since the generated framework is the same as the target name).

They share code, so having these duplicate targets doesn't increase the code maintenance burden. It's just ugly.